OZON.ru.js

```diff
@@ -484,119 +199,210 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
+				"date": "2001",
+				"ISBN": "9785845901255",
+				"abstractNote": "Те, кто еще не знакомы с вопросами объектно-ориентированного анализа и проектирования, наверняка планируют освоить эту область знаний. Данная книга станет хорошим путеводителем и позволит шаг за шагом пройти путь от определения требований к системе до создания кода. В книге рассматривается унифицированный язык моделирования UML, который является признанным стандартом для описания моделей и обеспечивает возможность общения между разработчиками. Для иллюстрации всего процесса объектно-ориентированного анализа и проектирования в книге приводится исчерпывающее описание реального примера. В нем показано, как перейти от этапа объектно-ориентированного проектирования к созданию кода на языке Java. Книга рассчитана на читателей с различным уровнем подготовки, интересующихся вопросами объектно-ориентированного анализа и проектирования.",
+				"language": "Русский",
+				"libraryCatalog": "OZON.ru",
+				"numPages": "496",
+				"publisher": "Вильямс",
+				"url": "http://www.ozon.ru/context/detail/id/117417/",
+				"attachments": [],
 				"tags": [],
-				"seeAlso": [],
-				"attachments": [
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.ozon.ru/context/detail/id/1335648/",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Искусство программирования. Том 1. Основные алгоритмы",
+				"creators": [
 					{
-						"title": "OZON.ru Link",
-						"mimeType": "text/html",
-						"snapshot": false
+						"firstName": "Дональд Эрвин",
+						"lastName": "Кнут",
+						"creatorType": "author"
```

Comment:
If I am matching this up correctly, looks like we lost a translator. Is that expected?